src/styles/lines/lines.js

```diff
@@ -237,6 +241,7 @@ Object.assign(Lines, {
         draw.width = StyleParser.createPropertyCache(draw.width, StyleParser.parseUnits);
         draw.next_width = StyleParser.createPropertyCache(draw.width, StyleParser.parseUnits); // width will be computed for next zoom
         draw.z = StyleParser.createPropertyCache(draw.z, StyleParser.parseUnits);
+        draw.offset = StyleParser.createPropertyCache(draw.offset || 0, StyleParser.parseUnits);
```

Comment:
If we are going to only accept `px` units (which I think we decided on), we should enforce that, probably here (first time the parameter is processed). Will need to double-check, but I think we should be able to do that by checking (if `draw.offset` isn't null) that `draw.offset.value.units === 'px'`, and if not, log a warning and setting `draw.offset` to null. We may also want to add the ability to specify different default units for `parseUnits` though, so that offset values can have pixels as default units (if we decide we want that).